### PR TITLE
landing page fix

### PR DESCRIPTION
**Pull request by Lharbeeb**
this is not completed yet, but i have all the required images inserted now

### CODE REVIEW

**Review by tiana0803 — COMMENTED** [review]
The Auth_wiki text on the Top Nav bar is different from the one at the bottom Nav bar
F57923CB-F35F-4B65-ADA2-26034C0DFBCC

The search icon is also a different shade from the text

**Comment by ClareJitu** [comment]
IMG_20220730_184113
The stroke of the search bar should be faint
The 'Developers' in the navigation bar should be 'Documentation' as seen in the design.


Please take note of Tiana's comment

**Comment by ClareJitu** [comment]
image
The 'I4G' in the footer section is absent, look closely.           
Also, the I4G just before the footer...just incase

**Comment by ClareJitu** [comment]
Also, the inner border of the navigation bar has a colour

**Comment by spykelionel** [comment]
> this is not completed yet, but i have all the required images inserted now

Okay. You can complete it then. This will be moved to In-progress section of the project. well done. Keep up

File: frontend/css/landing.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
-body {
```
Comment (spykelionel) [inline frontend/css/landing.css:1]: The body shouldn't `display: flex`
Check your page. It goes beyond the screen width

File: frontend/css/landing.css [hunk 2]
```diff
@@ -27,20 +27,20 @@ header,
 
 header .company-logo {
     font-size: 1.25rem;
-    letter-spacing: .05rem;    
+    letter-spacing: .05rem;
     text-align: left;
 }
 
 header a,
 header button {
```
Comment (spykelionel) [inline frontend/css/landing.css:35]: check base.css styles for button styles.

File: frontend/css/landing.css [hunk 3]
```diff
@@ -153,12 +153,170 @@ header input:focus {
...
+    width: 100%;
+}
+.middle5a{
+    display: flex;
+    flex-direction: column;
+}
+
+.middle6{
+    margin: 0 635px 94px 635px;
+}
```
Comment (spykelionel) [inline frontend/css/landing.css:319]: All class names should follow the naming convention.
`lp` for landing page

File: frontend/css/landing.css [hunk 4]
```diff
@@ -319,28 +477,28 @@ footer a:hover {
...
     header {
         margin: 0 -6.5rem;
         padding: 2rem 6.5rem;
     }
 
     header a,
-    header button {        
-        font-size: 1.1rem;        
+    header button {
+        font-size: 1.1rem;
```
Comment (spykelionel) [inline frontend/css/landing.css:491]: check base.css for button styles

File: frontend/css/landing.css [hunk 5]
```diff
@@ -319,28 +477,28 @@ footer a:hover {
...
     }
 
     header button {
-        min-width: 6rem;        
+        min-width: 6rem;
```
Comment (spykelionel) [inline frontend/css/landing.css:495]: check base.css for button styles

File: frontend/index.html [hunk 6]
```diff
@@ -6,6 +6,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <link rel="stylesheet" type="text/css" href="css/base.css">
         <link rel="stylesheet" type="text/css" href="css/style.css" />
+        <link rel="stylesheet" type="text/css" href="css/landing.css" />
```
Comment (spykelionel) [inline frontend/index.html:9]: use style.css in the css folder. This is the default CSS file for our landing page. You move your styles from landing.css to style.css

File: frontend/index.html [hunk 7]
```diff
@@ -17,10 +18,210 @@
     <header>
         <h2 class="company-logo">AUTH_WIKI</h2>
         <nav class="desktop_nav">
+
             <a href="">About</a>
             <a href="">Products</a>
             <a href="">Developers</a>
             <div class="search-bar">
+              <input type="text" placeholder="Search" />
```
Comment (spykelionel) [inline frontend/index.html:26]: check base.css for input styles

File: frontend/index.html [hunk 8]
```diff
@@ -17,10 +18,210 @@
...
+                  src="./images/undraw_developer_activity_re_39tg 3.png"
+                  alt=""
+                  id="image1"
+                />
+              </div>
+            </div>
+            <div class="middle1b">
+              <div class="middle1bi">
+                <a href="#">
+                  <img src="./images/Frame 291.png" alt="#" id="image2" />
```
Comment (spykelionel) [inline frontend/index.html:69]: follow class naming convention. `lp-` for landing-page

File: frontend/index.html [hunk 9]
```diff
@@ -17,10 +18,210 @@
...
+                  <!--  <p>
+                                Create dynamic and interactive web content like applications and
+                                browsers with Javascript.
+                            </p>
+                            <br />
+                            <p>Learn more...</p> -->
```
Comment (spykelionel) [inline frontend/index.html:75]: remove unused html

File: frontend/index.html [hunk 10]
```diff
@@ -17,10 +18,210 @@
...
+                </a>
+              </div>
+              <div class="middle1bii">
```
Comment (spykelionel) [inline frontend/index.html:78]: follow class naming convention. lp- for landing-page

File: frontend/index.html [hunk 11]
```diff
@@ -17,10 +18,210 @@
...
+                  <!-- <p>
+                                Build websites, software, automate tasks, and conduct data
+                                analysis with Python.
+                            </p>
+                            <br />
+                            <p>Learn more...</p> -->
+                </a>
+              </div>
+            </div>
+            <div class="middle1c">
```
Comment (spykelionel) [inline frontend/index.html:90]: follow class naming convention. lp- for landing-page

File: frontend/index.html [hunk 12]
```diff
@@ -17,10 +18,210 @@
...
+              <a href="#"><button>view all Strategies</button></a>
+            </div>
+          </div>
+          <div class="middle2">
+            <div class="middle2a">
```
Comment (spykelionel) [inline frontend/index.html:95]: follow class naming convention. lp- for landing-page

File: frontend/index.html [hunk 13]
```diff
@@ -17,10 +18,210 @@
...
+                                Build websites, software, automate tasks, and conduct data
+                                analysis with Python.
+                            </p>
+                            <br />
+                            <p>Learn more...</p> -->
+                </a>
+              </div>
+            </div>
+            <div class="middle1c">
+              <a href="#"><button>view all Strategies</button></a>
```
Comment (spykelionel) [inline frontend/index.html:91]: Avoid nesting buttons inside of a link. If it should be a link. Use the necessary styles for it.

File: frontend/index.html (hunk 12 above)
Comment (spykelionel) [inline frontend/index.html:95]: follow class naming convention. lp- for landing-page

File: frontend/index.html [hunk 14]
```diff
@@ -17,10 +18,210 @@
...
+              <h1>Contribute and Interact with Code Samples in our Community</h1>
+              <br />
+              <h3>
+                Find and contribute answers to technical challenges you and others
+                face when you code.
+              </h3>
+              <br />
+              <button>Join The Community</button>
+            </div>
+            <div class="middle2b">
```
Comment (spykelionel) [inline frontend/index.html:105]: follow class naming convention. lp- for landing-page

File: frontend/index.html [hunk 15]
```diff
@@ -17,10 +18,210 @@
...
+              <img src="./images/PRODUCT pythonf 2.png" alt="" />
+              <img src="./images/PRODUCT pythonf 1.png" alt="" />
+            </div>
+          </div>
+          <div class="middle3">
+            <div class="middle3a">
```
Comment (spykelionel) [inline frontend/index.html:111]: follow class naming convention. lp- for landing-page

File: frontend/index.html [hunk 16]
```diff
@@ -17,10 +18,210 @@
...
+                <img src="./images/Frame 235.png" alt="#" />
+              </div>
+            </div>
+            <div class="middle3c">
+              <button>View documentation</button>
+            </div>
+          </div>
+          <div class="middle4">
+            <div class="middle4a">
+              <div class="middle4ai">
```
Comment (spykelionel) [inline frontend/index.html:139]: follow class naming convention. lp- for landing-page

File: frontend/index.html [hunk 17]
```diff
@@ -17,10 +18,210 @@
...
+                <div>
+                  <img src="./images/Frame 29.png" alt="" />
+                </div>
+                <div>
+                  <h3>UNAUTHENTICATED</h3>
+                </div>
+              </div>
+              <div class="middle4aii">
```
Comment (spykelionel) [inline frontend/index.html:147]: follow class naming convention. lp- for landing-page

File: frontend/index.html [hunk 18]
```diff
@@ -17,10 +18,210 @@
...
+                <p>
+                  A Public Space to View The Platform and Basic Information About
+                  it.
+                </p>
+              </div>
+              <div class="middle4aiii"><button>Join Community</button></div>
+            </div>
+            <div class="middle4b">
+              <div class="middle4bi">
```
Comment (spykelionel) [inline frontend/index.html:156]: follow class naming convention. lp- for landing-page

File: frontend/index.html [hunk 19]
```diff
@@ -17,10 +18,210 @@
...
+                  <h3>AUTHENTICATED</h3>
+                </div>
+              </div>
+              <div class="middle4bii">
+                <p>
+                  Have Full Access to The Platform Contribute However Way, View Code
+                  Samples and be Able to Download Codes.
+                </p>
+              </div>
+              <div class="middle4biii"><button>Get Started</button></div>
```
Comment (spykelionel) [inline frontend/index.html:170]: follow class naming convention. lp- for landing-page

File: frontend/index.html [hunk 20]
```diff
@@ -17,10 +18,210 @@
...
+            <div class="middle4b">
+              <div class="middle4bi">
+                <div>
+                  <img src="./images/Frame 29.png" alt="" />
+                </div>
+                <div>
+                  <h3>AUTHENTICATED</h3>
+                </div>
+              </div>
+              <div class="middle4bii">
```
Comment (spykelionel) [inline frontend/index.html:164]: follow class naming convention. lp- for landing-page

File: frontend/index.html [hunk 21]
```diff
@@ -17,10 +18,210 @@
...
+                  Have Full Access to The Platform Contribute However Way, View Code
+                  Samples and be Able to Download Codes.
+                </p>
+              </div>
+              <div class="middle4biii"><button>Get Started</button></div>
+            </div>
+          </div>
+          <div class="middle5">
+            <div class="middle5a">
+              <h1>Team AUTH_WIKI</h1>
```
Comment (spykelionel) [inline frontend/index.html:175]: follow class naming convention. lp- for landing-page

File: frontend/index.html [hunk 22]
```diff
@@ -17,10 +18,210 @@
...
+              <p>
+                Our team is dedicated to building great software to solve the
+                problems our customers face everyday.
+              </p>
+              <a href="#"><button>Meet The Team</button></a>
+            </div>
+            <div class="middle5b">
```
Comment (spykelionel) [inline frontend/index.html:182]: follow class naming convention. lp- for landing-page

File: frontend/index.html [hunk 23]
```diff
@@ -17,10 +18,210 @@
...
+              <img
+                src="./images/undraw_team_collaboration_re_ow29 2.png"
+                alt="#"
+              />
+            </div>
+          </div>
+          <div class="middle6">
```
Comment (spykelionel) [inline frontend/index.html:189]: follow class naming convention. lp- for landing-page

**Review by spykelionel — CHANGES_REQUESTED** [review]
Great work. Keep pushing. 
follow class naming convention.` lp-` for landing-page
Please read other comments on the files as well. Thanks

**Comment by spykelionel** [comment]
@ClareJitu @tiana0803 @Lharbeeb Check your preview
https://deploy-preview-59--auth-wiki.netlify.app/

**Comment by ClareJitu** [comment]
Omor, please Lionel, can you go through the design on the figma board. What you sent us to go through is quite different from our designs.
Also, we sent our first set of reviews and we noticed you have not implemented them in your code.
The images you downloaded for the documentation section, some of them are different from our designs.
Please go through them. Thank you

**Comment by spykelionel** [comment]
> Omor, please Lionel, can you go through the design on the figma board. What you sent us to go through is quite different from our designs.
> Also, we sent our first set of reviews and we noticed you have not implemented them in your code.
> The images you downloaded for the documentation section, some of them are different from our designs.
> Please go through them. Thank you

@Lharbeeb check this.

**Comment by spykelionel** [comment]
@ClareJitu @Lharbeeb will take care of this. Thanks for your feedback.

**Pull request closed without merging**